src/main/java/core/basesyntax/service/operations/FruitTransactionCreationServiceImpl.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax.service.operations;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+
+public class FruitTransactionCreationServiceImpl implements FruitTransactionCreationService {
+    private FruitDao dao;
+
+    public FruitTransactionCreationServiceImpl(FruitDao dao) {
+        this.dao = dao;
+    }
+
+    @Override
+    public void createTransactions(List<String[]> data) {
+        FruitTransaction transaction;
+        FruitTransaction.Operation operation = null;
+        String fruit;
+        int amount;
+        for (String[] parameters : data) {
+            transaction = new FruitTransaction();
+            String operationValue = parameters[0];
+            String fruitType = parameters[1];
+            int fruitQuantity = Integer.parseInt(parameters[2]);
+            for (FruitTransaction.Operation op : FruitTransaction.Operation.values()) {
```

Comment:
You don't need loop here. You can use simple:
`FruitTransaction.Operation operation = FruitTransaction.Operation.get(operationValue)` cause you have a map with all operation.